src/hid-sensor.cpp

```diff
@@ -339,6 +351,34 @@ uint32_t hid_sensor::fps_to_sampling_frequency( rs2_stream stream, uint32_t fps
     else
         return fps;
 }
+void hid_sensor::set_imu_sensitivity( rs2_stream stream, float value ) 
+{
+    _imu_sensitivity_per_rs2_stream[stream] = value;
+}
+
+float hid_sensor::get_imu_sensitivity(rs2_stream stream)
+{
+    if( _imu_sensitivity_per_rs2_stream.find( stream ) != _imu_sensitivity_per_rs2_stream.end() )
+    {
+        return _imu_sensitivity_per_rs2_stream[stream];
+     }
+     else 
+        // gyro sensitivity default value is +-1000 therefore returning 30.5 resolution
+        // accel sensitivity default value is +-4g therefore returning 1.95 resolution
+        return stream == RS2_STREAM_GYRO ? 30.5f : 1.95f;
+}
+
+double hid_sensor::get_imu_sensitivity_converted( rs2_stream stream )
```

Comment:
Please add a comment to this function declaration about what it is used for? converting what to what?

Reply:
Done